pebbles/views/instances.py

```diff
@@ -320,18 +304,27 @@ def patch(self, instance_id):
         instance = Instance.query.filter_by(id=instance_id).first()
         if not instance:
             abort(404)
-
-        log_type = args['type']
-        if not log_type:
+        log_record_json = args['log_record_json']
+        if not log_record_json:
             abort(403)
+        log_record = json.loads(log_record_json)
 
-        if log_type in ('provisioning', 'deprovisioning'):
-            log_dir, log_file_name = self.get_base_dir_and_filename(
-                instance_id, log_type, create_missing_filename=True)
+        instance_log = InstanceLog(instance_id)
+        instance_log.log_type = log_record['log_type']
+        instance_log.log_level = log_record['log_level']
+        instance_log.timestamp = float(log_record['timestamp'])
+        instance_log.message = log_record['message']
```

Comment:
clamping either here or even more preferably in a setter in the InstanceLog model.